config/routes.rb

```diff
@@ -5,7 +5,8 @@
 
   namespace :api, defaults: { format: :json } do
     namespace :v1 do
```

Comment:
I'll see if I can find a good document on this. I should brush up on it myself. Frankly, you're doing exactly what I would do. This comment was more to let you know that. 😸